example-web/src/App.test.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
```

Comment:
This will obviously fail, so I wonder whether we should just remove the tests or go with the simple `toMatchSnapshot` test only to keep it vital.

Reply:
This is just the standard test generated by create-react-app. I'll remove it.